e2e_tests/.flake8

```diff
@@ -4,9 +4,57 @@ max-line-length = 100
 # We ignore F401 in __init__.py because it is expected for there to be
 # "unused imports" when defining a "regular" package. (This file is
 # implicitly executed when the package is imported, and the imports would
-# be used by the importer.) We ignore patch_saver_restore.py because it includes
-# a near-verbatim TensorFlow function with a small patch.
-per-file-ignores = __init__.py:F401 patch_saver_restore.py:E111,E114,
+# be used by the importer.)
+per-file-ignores =
+  __init__.py:F401,I2041
```

Comment:
I don't understand the `I*` ignore directives. They're `Innnn`, where according to the [README](https://github.com/atollk/flake8-import-restrictions) the rules are of the form `IMRnnn`. Why does this work? Are these the aliases we should be using for the rules in question?

Reply:
the author renamed it in 2.0, while we're using [1.1.1](https://github.com/atollk/flake8-import-restrictions/tree/v1.1.1) because v2.0 needs flake8 5.x (we have 3.x), and does not support python 3.7.